### ISSUE TITLE
Space at the end of keybinding not parsed

### ISSUE THREAD

**Issue by nfraprado** [vit-project/vit#211]
I'd like to have a keybinding that presses Space right at the end, but it doesn't seem to work.
This is the snippet of my keybindings where I'm trying this:
```
a = {ACTION_NOOP}
aa = {ACTION_TASK_ADD}
ai = aatype:in
an = aatype:next
```
I'd like to add a space at the end of the 'ai' line as well as at the end of the 'an' line. I have tried both a literal space and a `<Space>` but neither seem to work.
I'd like to do this so when I type those commands I don't have to manually add a space to add a task of the given type.
Is there a way to do this right now?

Also, I'm running the 2.0.0 version of vit.

**Comment by thehunmonkgroup** [comment]
Unfortunately, Python's ConfigParser module trims strings on the right hand side of the equals sign, so some workaround has to be put into place to solve this problem.

I chose to use the same workaround I did for these issues on the left side of the equals sign, namely, to declare some custom angle bracket tokens. See https://github.com/scottkosty/vit/commit/a6a48912a3e19d4bbadc5de5668f03652a187d0f for the fix and a listing of the special tokens.

Alternatively, this hack should work on 2.0.0 in the meantime:

```ini
ai = aatype:in< >
```

Basically, anything inside the angle brackets is lower-cased and passed as a full string instead of an individual key press, which in this case is a space.
